.github/workflows/validateChangedFiles.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
Missing some useful options:
```suggestion
set -o errexit -o nounset -o pipefail
```